src/test/java/com/williamfeliciano/sba_movies/services/MoviesServiceTests.java

```diff
@@ -0,0 +1,87 @@
+package com.williamfeliciano.sba_movies.services;
+
+import com.williamfeliciano.sba_movies.dtos.MovieDetailsDto;
+import com.williamfeliciano.sba_movies.dtos.MovieDto;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+public class MoviesServiceTests {
+
+    @Mock
```

Comment:
You don't need the Mock annotation.
The class you want to test is `MoviesService`. This class should be annotated with `@InjectMocks`. But this is only necessary when you have other dependencies inside `MoviesService` like repositories or other services. This is not the case for the moment (but keep it in mind, you will need it soon).